src/components/02-components/time-log-table/time-log-table.tsx

```diff
@@ -0,0 +1,119 @@
+import {
+  Box,
+  Button,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Typography,
+  useTheme,
+} from '@mui/material';
+import FileCopyIcon from '@mui/icons-material/FileCopy';
+import DeleteIcon from '@mui/icons-material/Delete';
+import EditIcon from '@mui/icons-material/Edit';
+
+export type TimeLogTableProps = {
+  timeLogInfo: {
+    dateTime: string;
+    hour: number;
+    goals: string;
+    learningOutcomes: string;
+    status: string;
+  }[];
+  cta: string;
+};
+
+export default function TimeLogTable({ timeLogInfo, cta }: TimeLogTableProps) {
+  const theme = useTheme();
+
+  // Style
+  const contentStyle = {
+    p: theme.spacing(5),
+  };
+
+  const headerWrapper = {
+    display: 'flex',
+    justifyContent: 'space-between',
+  };
+
+  const iconWrapperStyle = {
+    display: 'flex',
+  };
+
+  const iconStyle = {
+    mr: theme.spacing(1),
```

Comment:
The last icon shouldn't have a left margin. You can use `gap` here or remove the margin of the `:last-child` element.